core/server/fluxruntime_test.go

```diff
@@ -62,7 +55,12 @@ func TestGetReconciledObjects(t *testing.T) {
 		},
 	}
 
-	g.Expect(k.Create(ctx, &reconciledObj)).Should(Succeed())
+	k := fake.NewClientBuilder().
```

Comment:
:thinking: Not understanding why we need to move to this lib? Do we still get some of the same validation and RBAC stuff?

Reply:
I've opened #1895 without test improvements.